packages/dashboard/src/components/auth/tests/private-route.test.tsx

```diff
@@ -26,14 +28,18 @@ describe('PrivateRoute', () => {
     expect(component.html()).toMatchSnapshot();
   });
 
-  test('redirects when unauthenticated', () => {
-    mount(
-      <Router history={history}>
-        <Switch>
-          <PrivateRoute path="/private" exact />
-        </Switch>
-      </Router>,
-    );
+  test('redirects when unauthenticated', async () => {
+    await act(async () => {
+      mount(
+        <AuthenticatorContext.Provider value={new FakeAuthenticator()}>
+          <Router history={history}>
+            <Switch>
+              <PrivateRoute path="/private" exact />
+            </Switch>
+          </Router>
+        </AuthenticatorContext.Provider>,
+      );
+    });
```

Comment:
@vallq  here too, but as you can notice it has a context provider now.

Context: Val is replacing Enzyme for RTL.